package.json

```diff
@@ -3,30 +3,36 @@
   "private": true,
   "scripts": {
     "standardjs": "standard",
-    "stylelint": "stylelint **/*.scss",
-    "test": "jest"
+    "stylelint": "stylelint app/assets/stylesheets/**/*.scss",
```

Comment:
Stylelint path with glob issue: this only runs against sub-directories due to a path and glob problem, use `app/assets/stylesheets/` instead, this should target all files including sub-directories

Reply:
Done and tested with violations in nested and top level files 👍